pages/components/Grids/grids.html

```diff
@@ -0,0 +1,251 @@
+                                    <span>Add to cart</span>
+                                </button>
+                            </div>
+                        </div>
+                        <div class="card card-vt">
+                            <div class="card-header">
+                                <img class="card-img" src="/Assets/Images/card-image-3.jpg" alt="card image">
+                                <span class="btn-wishlist"><img src="/Assets/Icons/btn-wishlist.svg" alt="card icon"></span>
+                            </div>
+                            <div class="card-body">
+                                <h5 class="card-title">Apple</h5>
+                                <small class="card-subtitle">Apple airpods</small>
+                                <div class="card-pricing">
+                                    <h5>Rs. 1750</h5>
+                                    <h5 class="font-light strikethrough">Rs.3500</h5>
+                                    <h5 class="card-discount">(50% Off)</h5>
+                                </div>
+                            </div>
+                            <div class="card-footer">
+                                <button class=" btn btn-solid-primary">Buy Now</button>
+                                <button class="btn btn-icon btn-solid-secondary">
+                                    <img class="btn-icon-img" src="/Assets/icons/btn-cart.svg" alt="icon">
+                                    <span>Add to cart</span>
+                                </button>
+                            </div>
+                        </div>
+                        <div class="card card-vt">
+                            <div class="card-header">
+                                <img class="card-img" src="/Assets/Images/card-image-3.jpg" alt="card image">
+                                <span class="btn-wishlist"><img src="/Assets/Icons/btn-wishlist.svg" alt="card icon"></span>
+                            </div>
+                            <div class="card-body">
+                                <h5 class="card-title">Apple</h5>
+                                <small class="card-subtitle">Apple airpods</small>
+                                <div class="card-pricing">
+                                    <h5>Rs. 1750</h5>
+                                    <h5 class="font-light strikethrough">Rs.3500</h5>
+                                    <h5 class="card-discount">(50% Off)</h5>
+                                </div>
+                            </div>
+                            <div class="card-footer">
+                                <button class=" btn btn-solid-primary">Buy Now</button>
+                                <button class="btn btn-icon btn-solid-secondary">
+                                    <img class="btn-icon-img" src="/Assets/icons/btn-cart.svg" alt="icon">
```

Comment:
suggestion - In alt attribute do better naming so whenever IMG loads then it gives a prior idea of IMG.